src/background/communication.js

```diff
@@ -56,24 +58,26 @@ async function send(protocol) {
   await updateSentAt();
 }
 
-(async function () {
+observe('terms', async (terms) => {
```

Comment:
"terms" has the semantic of "terms accepted", right? If so, perhaps we can renamed it?

Reply:
yes, it means terms accepted, but I think it is easy to reason about, as `terms: true` means, you have them, which means they are accepted. Why would you add an additional suffix?

Reply:
OK. Reading through it, it was not obvious to me that it is a flag and what it means.